providers/multipart/src/test/java/org/jboss/resteasy/plugins/providers/multipart/Mime4JWorkaroundTest.java

```diff
@@ -0,0 +1,47 @@
+package org.jboss.resteasy.plugins.providers.multipart;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class Mime4JWorkaroundTest {
+
+   @Before
```

Comment:
Ideally, the test should leave the environment in the same state it was before the run. So we should probably store the value of the property before the execution and restore it afterwards.

Reply:
Good catch. -I'll add `@BeforeClass` and `@AfterClass` methods to ensure the state of the system property is the after the test as it was before the test. And the same for other test too.  Thanks!

Reply:
@asoldano I've added the before/afterClass methods to ensure the system properties are same before and after the test.  Can you give this PR another look?  Thanks!